source/user-manual/api/getting-started.rst

```diff
@@ -898,6 +898,35 @@ Adding an agent is now easier than ever. Simply send a request with the agent na
       "error": 0
     }
 
+Ingest security events
+^^^^^^^^^^^^^^^^^^^^^^
+
+.. versionadded:: 4.5.0
+
+It is possible to send security events to analyze using the Wazuh API directly.
+
+.. code-block:: console
+
+    # curl -k -X GET 'https://localhost:55000/events' -H  'Authorization: Bearer $TOKEN' -H  'Content-Type: application/json' -d '{"events": ["Event value 1", "{\"someKey\": \"Event value 2\"}"]}'
```

Comment:
```suggestion
    # curl -k -X POST "https://localhost:55000/events" -H  "Authorization: Bearer $TOKEN" -H  "Content-Type: application/json" -d '{"events": ["Event value 1", "{\"someKey\": \"Event value 2\"}"]}'
```
The method was wrong. Also, when using `'` instead of `"` to wrap the Atuh token, I get this error:
```
# curl -k -X POST 'https://localhost:55000/agents' -H 'Authorization: Bearer $TOKEN'
{"title": "Unauthorized", "detail": "Invalid token"}
```

```
# curl -k -X GET "https://localhost:55000/agents" -H "Authorization: Bearer $TOKEN"
{"data": {"affected_items": [{"os": {"arch": "x86_64", "codename": "Focal Fossa", "major": "20", "minor": "04", "name":...
```